.editorconfig

```diff
@@ -8,6 +8,11 @@ end_of_line = LF
 insert_final_newline = true
 trim_trailing_whitespace = true
 
+; 4-column tab indentation
+[*.{cs,yaml,lua,sh,ps1}]
+indent_style = tab
+indent_size = 4
+
 ; 4-column tab indentation and .NET coding conventions
 [*.cs]
 indent_style = tab
```

Comment:
Lines 17 to 19 are now redundant with the above.

Reply:
I'd say 16 to 20.